web-app/js/portal/cart/DownloadEstimator.js

```diff
@@ -30,6 +30,13 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
                 this._createDownloadEstimate(result, collection.getUuid(), callback);
             },
             failure: function(result, values) {
+                if (result.isTimeout) {
+                    var estimaterId = "downloadEst-" + collection.getUuid();
+                    var listOfElements = $('[id^='+estimaterId+']');
+                    if (listOfElements.length == 1) {
+                        listOfElements[0].innerHTML = '<div><span class="fa fa-fw fa-warning"></span>Unable to calculate download size.</div><div class="clear"></div>';
+                    }
+                }
                 this._createFailMessage(result, collection.getUuid());
```

Comment:
@bhasin85 
Ive not tested this but it seems that the fail message should be created by `this._createFailMessage`
`createFailMessage` is creating the html in the same way as success `createDownloadEstimate` which should work.

`createFailMessage` function should be fixed here